src/neo4j_graphrag/experimental/components/schema.py

```diff
@@ -72,28 +72,33 @@ class SchemaConfig(DataModel):
     """
 
     entities: Dict[str, Dict[str, Any]]
-    relations: Dict[str, Dict[str, Any]]
-    potential_schema: List[Tuple[str, str, str]]
+    relations: Optional[Dict[str, Dict[str, Any]]]
+    potential_schema: Optional[List[Tuple[str, str, str]]]
 
     @model_validator(mode="before")
     def check_schema(cls, data: Dict[str, Any]) -> Dict[str, Any]:
         entities = data.get("entities", {}).keys()
         relations = data.get("relations", {}).keys()
         potential_schema = data.get("potential_schema", [])
 
-        for entity1, relation, entity2 in potential_schema:
-            if entity1 not in entities:
+        if potential_schema:
+            if not relations:
```

Comment:
We could also create the list of relation from the potential schema.

Reply:
You can't include descriptions or specify properties for relations in the potential_schema though can you?

Reply:
No we can't, but if people are not providing a list of relationships, we can assume they do not care about description or properties.

Reply:
That's fine, I think we can create a new ticket for this though

Reply:
Yes sure, part of the things we need to decide about user experience.